src/box/schema.cc

```diff
@@ -293,6 +303,9 @@ sc_space_new(uint32_t id, const char *name,
 	space_cache_replace(NULL, space);
 	if (replace_trigger)
 		trigger_add(&space->on_replace, replace_trigger);
+	struct trigger *t = (struct trigger *) malloc(sizeof(*t));
+	trigger_create(t, on_replace_dd_system_space, NULL, (trigger_f0) free);
```

Comment:
The trigger should be allocated statically, like other DDL triggers.

Reply:
Other DDL triggers are declared statically and per system space. Here the same trigger is used for all system spaces. So in this case we have to create n (where n the number of system spaces) identical triggers. Otherwise I didn't get your suggestion...

Reply:
You're right. I forgot that we use inline lists. Let's leave it as is then.